data/Ertan222.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Ertan Taner",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Jr. Backend Developer | Animal Lover | Open Source Is Everything | Linux Enthusiast | Terminal > GUI ",
+  "avatar": "https://avatars.githubusercontent.com/u/62712555?v=4",
```

Comment:
change the avatar link to `https://github.com/Ertan222.png`

Reply:
Thanks for feedback, going to change it right now